src/core/namespace.c

```diff
@@ -1264,6 +1264,7 @@ int setup_namespace(
         _cleanup_(decrypted_image_unrefp) DecryptedImage *decrypted_image = NULL;
         _cleanup_(dissected_image_unrefp) DissectedImage *dissected_image = NULL;
         _cleanup_free_ void *root_hash = NULL;
+        _cleanup_free_ char *hash_device = NULL;
```

Comment:
i must say I find "hash_device" a surprising name for this, given that this is most likely a plain file no? Maybe call this verity_data or verity_image?

Reply:
I named it after what cryptsetup/veritysetup calls it, for cross-tools consistency - from "man veritysetup":

> open <data_device> <name> **<hash_device>** <root_hash> create <name> <data_device> **<hash_device>** <root_hash>

But I do not mind one way or another, so I'll change to verity_data

Reply:
they have a different focus though than we I'd say. This stuff we do here is clearly intended for files, they focus more on block devices. I think we should stick to more generic terminology hence?

Reply:
Sure, no problem